diffsync/__init__.py

```diff
@@ -175,11 +175,29 @@ def set_status(self, status: DiffSyncStatus, message: Text = ""):
         self._status = status
         self._status_message = message
 
+    @classmethod
+    def create_base(cls, diffsync: "DiffSync", ids: Mapping, attrs: Mapping) -> Optional["DiffSyncModel"]:
```

Comment:
With the latest changes I'm back to not seeing why we need to have these `_base` methods added at all. If the subclass overrides `create()` then by default this method won't be called unless they call `super().create()`, which is now exactly identical to calling `super().create_base()` - so why have two different methods that do the same thing?

Reply:
I can remove them if you really want but I know if would have solved a situation for one of my project we had 3 layers of classes, like this 
1. DiffSyncModel
2. NetworkImporterModel
3. CustomerDefinedNetworkImporterModel

In this scenario, the goal was to redefine `NetworkImporterModel.update()` with `CustomerDefinedNetworkImporterModel.update()` to change it's behavior so calling super() wasn't possible but we still have to call `DiffSyncModel.update()`
We solved it by calling `DiffSyncModel.update()` directly but I think this isn't ideal either. 
I think., having the base method provides a clean way to identify the code that must be executed.

Again, happy to remove if you prefer ... I'm not gonna fall on my sword for this one

Reply:
I actually ran into something similar with the ssot-servicenow plugin and the `create()` method, where I didn't want to call the immediate superclass's `create()` but still needed to call the base `DiffSyncModel.create()`. This would have been potentially useful there as well; I'm just not entirely persuaded that needing to do this is a sign of a good code smell. :-)

Reply:
Agreed, I think it would be redesign the API for 2.0 to manage this situation better